pkg/test/verify/cluster.go

```diff
@@ -25,9 +25,9 @@ func AddToolchainClusterAsMember(t *testing.T, functionToVerify FunctionToVerify
 	defer gock.Off()
 	status := test.NewClusterStatus(toolchainv1alpha1.ToolchainClusterReady, corev1.ConditionTrue)
 	memberLabels := []map[string]string{
-		Labels("", "", test.NameHost),
-		Labels(cluster.Member, "", test.NameHost),
-		Labels(cluster.Member, "member-ns", test.NameHost)}
+		Labels("", test.NameHost),
+		Labels("", test.NameHost),
```

Comment:
looks like a duplicated labels
```suggestion
		Labels("", test.NameHost),
```
and don't forget that we want to fail when the namespace is not set in the label.